lib/invoice.rb

```diff
@@ -14,4 +14,16 @@ def initialize(transaction_details)
     @created_at = transaction_details[:created_at]
     @updated_at = transaction_details[:updated_at]
   end
+
+  def update_id(id)
+    @id = id
```

Comment:
not sure if this is needed, it's not common to see ids change, unless it's an id linking to another model of data